tests/src/Regressions/coreclr/0014/avtest.csproj

```diff
@@ -16,7 +16,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <ReferenceLocalMscorlib>false</ReferenceLocalMscorlib>
     <CLRTestKind>BuildAndRun</CLRTestKind>
-    <CLRTestPriority>1</CLRTestPriority>
```

Comment:
Nit: You don't really need to specify 0, if CLRTestPriority is not specified, by default it is zero.
But do keep this around if you intend to downgrade it again